packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -52,9 +52,13 @@ export class EpochTracker implements IPersistedFileCache {
         protected readonly cache: IPersistedCache,
         protected readonly fileEntry: IFileEntry,
         protected readonly logger: ITelemetryLogger,
+        protected readonly cacheExpiryTimeoutOverrideMs: number | undefined,
     ) {
         // Limits the max number of concurrent requests to 24.
         this.rateLimiter = new RateLimiter(24);
+        if (cacheExpiryTimeoutOverrideMs !== undefined && cacheExpiryTimeoutOverrideMs >= defaultCacheExpiryTimeoutMs) {
```

Comment:
I'd prefer this to be soft failure. I.e. expiry to be adjusted down silently, maybe with some log statement. And mention that behavior in comments.